### ISSUE TITLE
Use the same region for equal images in Texture Atlas importer

### ISSUE THREAD

**Issue by guilhermefelipecgs** [godotengine/godot#28213]
**Godot version:**
1120b0b

**OS/device including version:**
5.0.7-arch1-1-ARCH

**Issue description:**
Just a suggestion, atlas pack could be even better if we don't repeat equals textures like this example above:
![Captura de tela de 2019-04-19 17-59-12](https://user-images.githubusercontent.com/1387165/56443883-dd782600-62cc-11e9-8602-80f4cffa7371.png)

**Comment by harrisyu** [comment]
Yes, I think that's the last step that we don't need external texture packer anymore.
